packages/sanity/src/structure/panes/document/statusBar/DocumentStatusBarActions.tsx

```diff
@@ -103,23 +103,20 @@ const DocumentStatusBarActionsInner = memo(function DocumentStatusBarActionsInne
                 />
               ) : (
                 <>
-                  {
-                    /** TODO DO WE STILL NEED THIS OR CAN WE MOVE THIS TO THE PLUGIN? */
-                    isBundleDocument(currentGlobalBundle) && formState?.value?._id ? (
-                      <BundleActions
-                        currentGlobalBundle={currentGlobalBundle}
-                        documentId={formState.value._id as string}
-                        documentType={documentType}
-                        {...actionProps}
-                        key={formState.value._id as string}
-                      />
-                    ) : (
-                      <div>
-                        {/* eslint-disable-next-line i18next/no-literal-string */}
-                        <Text>Not a bundle</Text>
-                      </div>
-                    )
-                  }
+                  {isBundleDocument(currentGlobalBundle) && formState?.value?._id ? (
```

Comment:
A tooltip related to the live edit is shown when hovering in the action.
<img width="346" alt="Screenshot 2024-09-06 at 10 14 18" src="https://github.com/user-attachments/assets/3541d8ba-1c27-40cc-8344-ad7f3629da6f">

Probably out of scope for this PR, but I think we should update how the document actions show and probably insert this action through the releases plugin. 
Will follow this in slack

Reply:
To keep this updated: we discussed this as part of a larger change and out of scope for this PR